src/core/include/openvino/op/constant.hpp

```diff
@@ -68,7 +68,7 @@ class OPENVINO_API Constant : public Op {
     /// \param values A vector of literals for initializing the tensor constant. The
     ///               size of values must match the size of the shape.
     template <typename T>
-    Constant(const element::Type& type, const Shape& shape, const std::vector<T>& values)
+    Constant(const element::Type_t& type, const Shape& shape, const std::vector<T>& values)
```

Comment:
What is  benefit of change public function API?

In many case the element passed here is result of `get_element_type()` which return reference. What this change do in that case the reference will be converted to `element::Type_t` stored locally and then provide reference to the function, then inside when `fill_data` call the enum value will be converted to `element::Type` class which result creation of this object and pass reference.

This change increase binary size by 1.5 KiB and I don't see benefit of it. (probably more correct would be to pass `element::Type_t` value, but other functions should be aligned to use same type.

I think this change can be reverted.

Reply:
Fixed